doc/source/Tutorials/Scripting/GettingStarted/index.md

```diff
@@ -1,152 +1,326 @@
+![The Sphere node in the Viewer](images/viewerSphere.png "The Sphere node in the Viewer")
 
+> Tip :
+> When using the _Script Editor_, modules only need to be imported once per session.
+
+
+### Changing plug values ###
+
+A plug's value can be returned with the `getValue()` method and modified with the `setValue()` method.
+
+Use the output of the _Script Editor_ to test the Sphere node's current Radius plug value:
+
+```python
+mySphere["radius"].getValue() # 1.0
 ```
-# Setting leaf plugs individually
-script["Sphere"]["transform"]["translate"]["x"].setValue( 1 )
-script["Sphere"]["transform"]["translate"]["y"].setValue( 2 )
-script["Sphere"]["transform"]["translate"]["z"].setValue( 3 )
 
-# Setting them all in one shot
-script["Sphere"]["transform"]["translate"].setValue( imath.V3f( 1, 2, 3 ) )
+Next, set the values of the sphere's Name, Radius, and Theta Max plugs:
+
+```python
+mySphere["name"].setValue( "mySphere" )
+mySphere["radius"].setValue( 3 )
+mySphere["thetaMax"].setValue( 180 )
 ```
 
-> Tip : The second form is often more convenient because it is shorter, and because the `V3f` class
-> provides lots of useful functionality for doing maths with vector values. As a newcomer to Gaffer
-> though, it's unlikely that we could have guessed the right syntax. Here drag and drop comes to our
-> rescue again - _Shift+Left Dragging_ the plug label from the NodeEditor and into the ScriptEditor
-> will enter the code for current value of the plug, providing a convenient syntax reference.
+![The Sphere node with adjusted plugs in the Node Editor](images/nodeEditorSphere.png "The Sphere node with adjusted plugs in the Node Editor")
 
-Making connections
-------------------
+![The Sphere node with adjusted plugs in the Viewer](images/viewerSpherePlugs.png "The Sphere node with adjusted plugs in the Viewer")
 
-As well as having values, plugs can be connected together so that values flow from one to another. We'll use the ScriptEditor to create a small node network to demonstrate this.
+> Note :
+> A scene node's Name plug determines the name of the object as it will appear in the scene hierarchy.
 
+
+### Setting compound plug values ###
+
+When a plug has its own child plugs, it is called a **compound plug**. An example would be the sphere's Transform plug, which has Translate, Rotate, and Scale child plugs. Plugs with a list of values are also compound plugs. The Rotate plug, for instance, contains a list with 3 values, each for a different axis. You can either `setValue()` each list value individually, or all at once using the V3f type.
```

Comment:
`list` is a specific data type in Python, so this is giving the impression that the plug stores a python list, which is not the case. The Rotate plug doesn't contain a list with 3 values. Instead it contains three child plugs, each with their own value (one for each axis). The "all at once" methods are just conveniences that call `setValue()` or `getValue()` on each child in turn. Like the dictionary stuff I mentioned on the other PR, this is a spot where I think it'd help to tighten up the language a little.